ResearchKit/Common/ORKHealthAnswerFormat.m

```diff
@@ -312,4 +316,8 @@ + (BOOL)supportsSecureCoding {
     return YES;
 }
 
+- (NSString *)stringForAnswer:(id)answer {
+    return [self.impliedAnswerFormat stringForAnswer:answer];
```

Comment:
If you can implement this in `ORKAnswerFormat`, then you don't need to this override here.
```
- (NSString*)stringForAnswer:(id)answer {
    ORKAnswerFormat *impliedFormat = [self impliedAnswerFormat];
    if (impliedFormat == self) {
        return nil;
    } else {
        return [impliedFormat stringForAnswer:answer];
    }
}
```